src/main/java/uk/gov/companieshouse/web/accounts/service/smallfull/impl/LoansServiceImpl.java

```diff
@@ -82,10 +84,15 @@
 
         String uri = LOANS_URI.expand(transactionId, companyAccountsId).toString();
 
+        if(StringUtils.isBlank(addOrRemoveLoans.getLoanToAdd().getDirectorName()) || addOrRemoveLoans.getLoanToAdd().getDirectorName().equals("Prefer not to say")) {
```

Comment:
This string should probably be a constant

Reply:
Changed the string to constant